### PR TITLE
perf(ui): improve bottom navigation ui by minimize the icon size

### PR DESCRIPTION
**Pull request by RobyCigar**
Closes 

## Description
I reduced the icon size by 0.5rem to improve the UI and make it more clean because I think previous size is too big. 

Before
Screenshot_2021-08-09_20-07-39
After
Screenshot_2021-08-09_20-07-55

Dont worry I already tested it
Screenshot_2021-08-09_20-06-13

## Current Tasks

- [ ] 
- [ ] 
- [ ]

### CODE REVIEW

**Comment by RobyCigar** [comment]
I think I made a mistake on this one, im so sorry.

File: components/layout/__tests__/navigation.test.tsx [hunk 1]
```diff
@@ -109,15 +109,15 @@ describe("Navigation", () => {
               </a>
             </li>
             <li
-              class="relative"
+              class="relative p-4"
```
Comment (RobyCigar) [inline components/layout/__tests__/navigation.test.tsx:112]: Please delete p-4

File: components/layout/__tests__/navigation.test.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ describe("Navigation", () => {
...
```
Comment (RobyCigar) [inline components/layout/__tests__/navigation.test.tsx:52]: please delete p-4 thank you

**Comment by RobyCigar** [comment]
It's not a big bug actually, I just accidentally add padding on line 112 and 52 please take a look if you're interested with my pr @resir014  
 thank you

**Comment by resir014** [comment]
This is not our priority right now. Please [pick another issue](https://github.com/kawalcovid19/wargabantuwarga.com/issues) to submit a pull request to.

**Pull request closed without merging**